modules/op-uninstalling-the-pipelines-operator.adoc

```diff
@@ -6,10 +6,18 @@
 [id='op-uninstalling-the-pipelines-operator_{context}']
 = Uninstalling the {pipelines-title} Operator
 
+You can uninstall the {pipelines-title} Operator in web console.
+
+[CAUTION]
+====
+Uninstalling only the Operator does not remove the {pipelines-shortname} default and optional components. If you uninstall the Operator without removing the CRs of optional components, you cannot remove them later.
+====
```

Comment:
Why did you add this note? Something similar is already on the page so now it is there twice. I don't see much value.

Reply:
@ppitonak  It was based on the peer review suggestion to add any warnings about what users should keep in mind before uninstalling. The idea is that some users might not read the full page from top to bottom. However, I agree with you that since this is pretty short chapter, maybe in this case it is okay to not mention it twice.